src/ontology/mondo-edit.obo

```diff
@@ -202099,12 +202100,13 @@ xref: HP:0002754 {source="MONDO:otherHierarchy"}
 xref: ICD10CM:M86.3 {source="Orphanet:324964", source="DOID:0060645", source="Orphanet:324964/e"}
 xref: MESH:C535456 {source="MONDO:equivalentTo"}
 xref: NCIT:C119042 {source="MONDO:equivalentTo"}
-xref: OMIM:259680 {source="Orphanet:324964", source="MONDO:equivalentTo", source="Orphanet:324964/e"}
+xref: OMIMPS:609628 {source="MONDO:equivalentTo"}
```

Comment:
I think this term needs to be split too.

Reply:
We don't want to lose the OMIMM classes, like OMIM:259680. And these xrefs are not all equiv to the OMIMPS

Reply:
I reviewed the x-refs, and all of them refer to the more generic term (ie not associated with a gene)